pkgs/by-name/cc/cctools/meson.build

```diff
@@ -0,0 +1,606 @@
+        'libstuff/args.c',
+        'libstuff/best_arch.c',
+        'libstuff/breakout.c',
+        'libstuff/bytesex.c',
+        'libstuff/checkout.c',
+        'libstuff/code_directory.c',
+        'libstuff/coff_bytesex.c',
+        'libstuff/crc32.c',
+        'libstuff/depinfo.c',
+        'libstuff/diagnostics.c',
+        'libstuff/dylib_roots.c',
+        'libstuff/dylib_table.c',
+        'libstuff/errors.c',
+        'libstuff/execute.c',
+        'libstuff/fatal_arch.c',
+        'libstuff/fatals.c',
+        'libstuff/get_arch_from_host.c',
+        'libstuff/get_toc_byte_sex.c',
+        'libstuff/guess_short_name.c',
+        'libstuff/hash_string.c',
+        'libstuff/hppa.c',
+        'libstuff/llvm.c',
+        'libstuff/lto.c',
+        'libstuff/macosx_deployment_target.c',
+        'libstuff/ofile.c',
+        'libstuff/ofile_error.c',
+        'libstuff/ofile_get_word.c',
+        'libstuff/print.c',
+        'libstuff/reloc.c',
+        'libstuff/rnd.c',
+        'libstuff/seg_addr_table.c',
+        'libstuff/set_arch_flag_name.c',
+        'libstuff/swap_headers.c',
+        'libstuff/symbol_list.c',
+        'libstuff/unix_standard_mode.c',
+        'libstuff/version_number.c',
+        'libstuff/vm_flush_cache.c',
+        'libstuff/write64.c',
+        'libstuff/writeout.c',
+        'libstuff/xcode.c'
+    ]
+)
+
+
+# Binaries
+ar = executable(
+    f'@target_prefix@ar',
+    include_directories : incdirs,
+    install : true,
+    link_with : [libstuff],
+    sources : [
+        'ar/append.c',
+        'ar/ar.c',
+        'ar/archive.c',
+        'ar/contents.c',
+        'ar/delete.c',
+        'ar/extract.c',
+        'ar/misc.c',
+        'ar/move.c',
+        'ar/print.c',
+        'ar/replace.c'
+    ]
+)
+install_man(
+    'ar/ar.1',
+    'ar/ar.5'
+)
+
+as = executable(
+    f'@target_prefix@gas',
+    include_directories : incdirs,
+    install : true,
+    link_with : [libstuff],
+    sources : ['as/driver.c']
+)
+
+as_common = files(
+    'as/app.c',
+    'as/as.c',
+    'as/atof-generic.c',
+    'as/atof-ieee.c',
+    'as/dwarf2dbg.c',
+    'as/expr.c',
+    'as/fixes.c',
+    'as/flonum-const.c',
+    'as/flonum-copy.c',
+    'as/flonum-mult.c',
+    'as/frags.c',
+    'as/hash.c',
+    'as/hex-value.c',
+    'as/input-file.c',
+    'as/input-scrub.c',
+    'as/layout.c',
+    'as/messages.c',
+    'as/obstack.c',
+    'as/read.c',
+    'as/sections.c',
+    'as/symbols.c',
+    'as/write_object.c',
+    'as/xmalloc.c'
+)
+
+as_arm = executable(
+    'as-arm',
+    c_args : [
+        '-DARM',
+        '-DNeXT_MOD'
+    ],
+    include_directories : [
+        incdirs,
+        'as',
+        'include/gnu'
+    ],
+    install : true,
+    install_dir : 'libexec/as/arm',
+    link_with : [libstuff],
+    sources : [as_common, 'as/arm.c']
+)
+
+as_i386 = executable(
+    'as-i386',
+    c_args : [
+        '-DI386',
+        '-Di486',
+        '-Di586',
+        '-Di686',
+        '-DNeXT_MOD'
```

Comment:
I didn't see `NeXT_MOD` in https://github.com/apple-oss-distributions/cctools/blob/main/cctools.xcodeproj/project.pbxproj

Reply:
It’s defined in https://github.com/apple-oss-distributions/cctools/blob/main/xcode/public_assembler.xcconfig.